server/src/main/java/org/candlepin/resource/OwnerResource.java

```diff
@@ -689,6 +687,36 @@ public void deleteLogLevel(@PathParam("owner_key") String ownerKey) {
         return page.getPageData();
     }
 
+    private List<ConsumerType> findAndValidateTypeLabels(
```

Comment:
Agree. I did it, but in ConsumerResource was same code for validation consumerTypeLabels, so I created ConsumerTypeValidator class and moved logic there, but I am still not satisfied with this because I think this should be also injected by guice instead creating new instance in constructor, what do you think?